be/test/runtime/mem_limit_test.cpp

```diff
@@ -23,89 +23,43 @@
 
 namespace doris {
 
-TEST(MemTestTest, SingleTrackerNoLimit) {
-    MemTracker t(-1);
-    EXPECT_FALSE(t.has_limit());
-    t.Consume(10);
-    EXPECT_EQ(t.consumption(), 10);
-    t.Consume(10);
-    EXPECT_EQ(t.consumption(), 20);
-    t.Release(15);
-    EXPECT_EQ(t.consumption(), 5);
-    EXPECT_FALSE(t.LimitExceeded(MemLimit::HARD));
-}
+class MemTrackerTest : public testing::Test {
+public:
+    MemTrackerTest() {}
+};
```

Comment:
Coz you are using `TEST`  only, so this class is not necessary?

Reply:
Yes, I will remove it